.github/workflows/generateChangelog.yml

```diff
@@ -0,0 +1,32 @@
+name: Generate Changelog
+
+on:
+  workflow_call:
+
+jobs:
+  generate_changelog:
+    name: 'Generate Changelog'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: 'develop'
+          ssh-strict: false
+          token: ${{ secrets.IDEE_GH_TOKEN }}
+          fetch-depth: '0'
```

Comment:
are the ssh and fetch-depth values required?  I didn't think we used those in other places so just curious why it's different here.

Reply:
I followed what is defined in createReleaseBranch.yml for ssh-strict.

fetch-depth is necessary. During my test, I found there are always errors [here](https://github.com/forcedotcom/salesforcedx-vscode/blob/7a03ff034d4d3983744d7c430230e7eb0dcacc78/scripts/create-release-notes.ts#L40).
Adding this line could fetch all history for all branches and tags. And the issue got fixed

Reply:
cool.  Def would be good to add a comment (can we do that in yml files?) about why it's necessary.  It makes sense that it's running out of room to get the required information (sorta...why don't we hit this limitation locally)

Reply:
Done!